.connector-store/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "OAuth2 DPoP Connector",
+    "owner": "WSO2",
+    "category": "OAuth 2.0 grant type",
+    "documentationUrl": "https://github.com/wso2-extensions/identity-oauth-dpop/blob/main/README.md",
+    "description": "DPoP ( Demonstrating Proof of Possession ) is an additional security mechanism for the token generation which overcomes the issue of bearer tokens which will not validate between who is requesting the token and who is actually using the token for access to a particular resource. The specification defines a mechanism to prevent illegal API calls from succeeding only with a stolen access token.",
```

Comment:
```suggestion
    "description": "DPoP (Demonstrating Proof of Possession) is an additional security mechanism for the token generation which overcomes the issue of bearer tokens which will not validate between who is requesting the token and who is actually using the token for access to a particular resource. The specification defines a mechanism to prevent illegal API calls from succeeding only with a stolen access token.",
```